projects/storefrontlib/src/lib/store-finder/components/store-finder-grid/store-finder-grid.component.html

```diff
@@ -1,15 +1,23 @@
-<ng-container *ngIf="!(isLoading$ | async) && locations; else loading">
-  <ul class="stores-grid">
-    <li
-      *ngFor="let location of locations?.pointOfServices; let i = index"
-      (click)="viewStore(location)"
-      id="{{ 'item-' + i }}"
-    >
-      <cx-store-finder-list-item
-        [location]="location"
-      ></cx-store-finder-list-item>
-    </li>
-  </ul>
+<ng-container
+  *ngIf="
+    !(isLoading$ | async) && (locations$ | async) as locations;
+    else loading
+  "
+>
+  <div class="container">
+    <div class="row">
+      <div
+        class="col-md-3"
+        *ngFor="let location of locations?.stores; let i = index"
+        (click)="viewStore(location)"
+        id="{{ 'item-' + i }}"
```

Comment:
template string here maybe instead of concatenation?

Reply:
I removed `id` attribute, seems to be not necessary.